application/contributors-list.jsx

```diff
@@ -0,0 +1,98 @@
+
+import React from "react";
+import Spinner from "../components/spinner.jsx";
+
+require("isomorphic-fetch");
+let utils = require("./utils")
+,   pp = utils.pathPrefix()
+;
+
+export default class ContributorsList extends React.Component {
+    constructor (props) {
+        super(props);
+        this.state = {
+            status:                     "loading"
+        ,   owner:                      null
+        ,   shortName:                  null
+        ,   substantiveContributors:    []
+        ,   nonSubstantiveContributors: []
```

Comment:
I'm not sure there is any value in listing non-substantive contributors at this stage, so I would drop it from the UI

Reply:
I've removed the non-substantive contributors from the UI